modules/ossm-performance-scalability.adoc

```diff
@@ -0,0 +1,79 @@
+////
+This module included in the following assemblies:
+- /v2x/customizing-installation-ossm.adoc
+////
+
+[id="ossm-performance-scalability_{context}"]
+= Performance and scalability
+
+{ProductName} strives to provide features with minimal resource overhead and aims to support very large meshes with high request rates while adding minimal latency.
+
+== Performance summary for Istio 1.6.8
```

Comment:
I wanted to specify the version partly so that if it doesn't match the product version that's clear to customers and partly so that we remember to go back an update this content for each new release.

Reply:
I hear that. 

I think that for me, and I think it didn't come to me clearly until my other comment about this further down which is related, is that referencing Istio 1.6.8 in the heading sounds more like the rest of the section is going to be about Istio. When I read that, the first question in my head was, okay, this is the performance summary for Istio, which I know is related, but I came here to learn about OSSM.

Our documentation should be a reflection of what we are asserting the product does. I think it would be better to make the heading `Performance summary`, to convey the idea that it's a performance summary about OSSM and then we could mention Istio 1.6.8 in the paragraph text.